ios/FluentUI/Core/Colors.swift

```diff
@@ -367,117 +368,127 @@ public final class Colors: NSObject {
 
     // MARK: Base semantic
 
-    @objc public static let background1 = UIColor(light: .white, dark: .black, darkElevated: gray900)
-    @objc public static let background1b = UIColor(light: .white, dark: gray950, darkElevated: gray800)
-    @objc public static let background1c = UIColor(light: .white, dark: gray900, darkElevated: gray800)
-    @objc public static let background2 = UIColor(light: gray25, dark: gray950, darkElevated: gray700)
-    @objc public static let background2b = UIColor(light: gray25, dark: gray700)
-    @objc public static let background3 = UIColor(light: gray50, dark: gray900, darkElevated: gray700)
-    @objc public static let background3b = UIColor(light: gray50, dark: gray600)
-    @objc public static let disabled = UIColor(light: gray100, dark: gray600)
-    @objc public static let foreground1 = UIColor(light: gray900, dark: gray100)
-    @objc public static let foreground1b = UIColor(light: gray900, dark: gray400)
-    @objc public static let foreground2 = UIColor(light: gray500, dark: gray400)
-    @objc public static let foreground2b = UIColor(light: gray500, dark: gray300)
-    @objc public static let foreground2c = UIColor(light: gray500, dark: gray500)
-    @objc public static let foreground2d = UIColor(light: gray500, dark: gray100)
-    @objc public static let foreground3 = UIColor(light: gray400, dark: gray400)
-    @objc public static let foreground3b = UIColor(light: gray400, dark: gray500)
-    @objc public static let foreground3c = UIColor(light: gray400, dark: gray600)
-    @objc public static let foreground4 = UIColor(light: gray300, dark: gray600)
-    @objc public static let foreground4b = UIColor(light: gray300, dark: gray500)
-    @objc public static let foreground5 = UIColor(light: gray200, dark: gray500)
-    @objc public static let foreground6 = UIColor(light: gray100, dark: gray700, darkElevated: gray600)
-    @objc public static let foreground6b = UIColor(light: gray100, dark: gray500)
-    @objc public static let foreground6c = UIColor(light: gray100, dark: .clear)
-    @objc public static let foreground7b = UIColor(light: .white, dark: gray100)
+    @available(*, deprecated, renamed: "surfaceDisabled")
+    @objc public static let disabled: UIColor = textDisabled
```

Comment:
The deprecated says it's renamed as surfaceDisabled, but we're setting it to textDisabled here. Am I missing something?

Reply:
thank's for catching that. yeah it forgot to update it.